### PR TITLE
[feature-1] First feature implemented

### PR DESCRIPTION
**Pull request by acctodownload**
Changes
- Added about.txt file.
- Modified Read.me.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # pull-request-demo
 
 Changes made in main branch.
+Changes done in feature-1 branch.
```
Comment (abhiraut041) [inline README.md:4]: Thanks for adding this line.

File: about.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Working on Pull Requests.
+New changes made locally.
```
Comment (abhiraut041) [inline about.txt:2]: Good Optimization changes.

File: README.md (hunk 1 above)
Comment (acctodownload) [reply README.md:4]: Thanks

File: about.txt (hunk 2 above)
Comment (acctodownload) [reply about.txt:2]: Thank you

**Review by abhiraut041 — APPROVED** [review]

**Pull request merged**